src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -419,6 +437,29 @@ private UserSettings LoadSettings()
             return settings;
         }
 
+        private void AddPackagesErrorBar()
```

Comment:
It's not just changing visibility, we also initialize `VSSolutionManager` here if it hasn't been initialized yet, and also update the error message. I don't think there's any issue with setting it to top here since it's a dynamic bar and besides, we already do it for missing packages or restart bar which have been working fine.